### DISCUSSION TITLE
frank-blog/2023/12/06/%E5%87%BD%E6%95%B0%E5%BC%8F%E7%BC%96%E7%A8%8B/

### DISCUSSION THREAD

**Discussion by giscus** [15666795711/frank-blog#4] [Announcements]
# frank-blog/2023/12/06/%E5%87%BD%E6%95%B0%E5%BC%8F%E7%BC%96%E7%A8%8B/

概述 介绍函数式编程的基本概念、优势以及在项目中的应用 目标：带领大家认识函数式编程，了解函数式编程的基本使用规则，能够在项目中应用，拓展大家思考问题的方向 问题： 论证用函数式编程的好处在哪儿 比封装普通函数优势在哪儿 对加入新功能和修改bug的提效是否有帮助 函数式编程 - 代码逻辑复用相关 react中的函数式编程    函数式编程介绍我们常见的编程范式有哪些 💡 特征，优缺点，在项目中的

http://localhost:4000/frank-blog/2023/12/06/%E5%87%BD%E6%95%B0%E5%BC%8F%E7%BC%96%E7%A8%8B/

**Comment by FrankJingZhi** [comment]
沙发

**Comment by kyangky** [comment]
牛哇
